test/outputpathoption.test.js

```diff
@@ -0,0 +1,30 @@
+const test = require('ava');
+const path = require('path');
+const fs = require('fs-extra');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const WebappWebpackPlugin = require('../');
+
+const { logo, generate, mkdir, compare, expected } = require('./util');
+
+test.beforeEach(async t => t.context.root = await mkdir());
+
+test('should allow for overriding the output path of favicons', async t => {
+  const customOutputPath = 'test/path';
+  const dist = path.join(t.context.root, 'dist');
+
+  await generate({
+    context: t.context.root,
+    output: {
+      path: dist,
+      publicPath: '/public/path',
+    },
+    plugins: [
+      new HtmlWebpackPlugin(),
+      new WebappWebpackPlugin({ logo, outputPath: customOutputPath }),
```

Comment:
Minor: any reason why not inlining `/test/path` here?

Reply:
Ha good catch I was actually trying to write tests without really looking into how all the utils worked so I had it used in multiple places at one point. Will update this